osu.Game/Screens/Play/HUD/LegacyComboSplash.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Skinning;
+
+namespace osu.Game.Screens.Play.HUD
+{
+    public class LegacyComboSplash : CompositeDrawable, ISkinnableDrawable
+    {
+        public Bindable<int> Current { get; } = new BindableInt { MinValue = 0 };
+
+        public bool UsesFixedAnchor { get; set; }
+
+        private Random random = new Random();
+
+        public LegacyComboSplash()
+        {
+            AlwaysPresent = true;
```

Comment:
This `AlwaysPresent` should not be required. Setting it on the created sprites should suffice.